Guide/Guide.md

```diff
@@ -0,0 +1,802 @@
+**QwMockData Root Guide**\
+May 23, 2024\
+Version 0.4
+
+If you have any questions or suggestions for the guide, please send an
+email to:\
+Ryan Conaway\
+mrc@jlab.org
+
+Running QwMockDataGenerator
+===========================
+
+Using the JAPAN-MOLLER repository, run the following commands in a
+terminal:
+
+        $  build/qwmockdatagenerator -r 4 -e 1:20000 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data <Data>
+        $  build/qwparity -r 4 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data-handler mock_datahandlers.map \
+           --data <Data> --rootfiles <Rootfiles>
```

Comment:
Please correct the "--data-handler" flag to be "--datahandlers".

Reply:
I corrected the typo in the --datahandlers argument.